Demo/Controller.cs

```diff
@@ -90,7 +91,7 @@ private string FormatException(Exception e)
 
         [HttpPost]
         [Route("/makeCredential")]
-        public async Task<JsonResult> MakeCredential([FromBody] AuthenticatorAttestationRawResponse attestationResponse)
+        public async Task<JsonResult> MakeCredential([FromBody] AuthenticatorAttestationRawResponse attestationResponse, CancellationToken cancellationToken)
```

Comment:
When is this cancellationToken called?

Reply:
When is HTTP request canceled (close tab, canceled XHR, ...)

Reply:
Having a cancellation token for a user action makes sense, not so sure about the MDS data load though, that should happen at startup and should not really be a cancellable event.

Reply:
Understood. I can remove it from here.

Anyway, a fired CancellationToken means no one is listening to the response. So it is meaningless to continue with request processing.

Reply:
@aseigler While not as usable, there are still situations that can occure that would benefit, at least in theory. Process wants to abort startup, e.g. fast restart, io hang etc. So best practice is to expose optional CancellationTokens in all async APIs.